### DISCUSSION TITLE
Does Sundance/Jacuzzi use Balboa controllers?

### DISCUSSION THREAD

**Discussion by techoguy** [netmindz/balboa_GL_ML_spa_control#16] [Q&A]
I was looking into ways to control my spa with Home assistant and found this project. But I can't find details about Sundance/Jacuzzi spa control boards specifically. I thought I read somewhere that Balboa actually makes these boards but can't find confirmation. My control board has an 8 pin like this project and an RJ45 connector next to it (not connected). The model number is 6600-881 Sundance 780 Series Control Panel. By chance does anyone know if this is indeed a Balboa board? At some point i'm hoping to see if the pins and signals are similar to what is documented here but I'm not at this location often. 
![image](https://user-images.githubusercontent.com/7158936/214472951-5b87fca7-0aa9-44db-b2df-9fcbb2b3c6ac.png)

**Comment by netmindz** [comment]
I've had a quick search of a few different spare parts companies to see if I could find any reference to Sundance top panels being cross-compatible with Balboa, but everything I found would suggest that they are just two totally different systems.

If you were to try your own reverse engineering, the first step is to use an oscilloscope to try and establish what the function of each pin is, then trying to decode the data stream. Not for the faint hearted!

**Reply by techoguy** [reply]
Okay thanks. I think a service tech had told me the Balboa wifi modules work with the newer Sundance and I know Sundance (Jacuzzi) white label a lot of stuff and do their best to hide it. I'll have to use an oscope to see if the data signals match either this project or the other in hopes that it does.
